.packit.yaml

```diff
@@ -0,0 +1,50 @@
+specfile_path: packaging/leapp.spec
+
+upstream_project_url: https://github.com/oamg/leapp
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  metadata:
+    owner: "@oamg"
+    project: leapp
+    targets:
+    - epel-7-ppc64le
+    - epel-7-x86_64
+    - epel-8-x86_64
+    - fedora-33-aarch64
+    - fedora-33-x86_64
+    - fedora-34-aarch64
+    - fedora-34-x86_64
+    - fedora-rawhide-aarch64
+    - fedora-rawhide-x86_64
+  actions:
+    post-upstream-clone:
+    # builds from PRs should have lower NVR than those from master branch
+    - sed -i "s/1%{?dist}/0%{?dist}/g" packaging/leapp.spec
+    get-current-version:
+    # get version from spec file instead from git tag
+    - bash -c "grep -m1 '^Version:' packaging/leapp.spec | grep -om1 '[0-9].[0-9.]**'"
+- job: copr_build
+  trigger: commit
+  metadata:
+    branch: master
+    owner: "@oamg"
+    project: leapp
+    targets:
+    - epel-7-ppc64le
+    - epel-7-x86_64
+    - epel-8-x86_64
+    - fedora-33-aarch64
+    - fedora-33-x86_64
+    - fedora-34-aarch64
+    - fedora-34-x86_64
+    - fedora-rawhide-aarch64
+    - fedora-rawhide-x86_64
```

Comment:
```suggestion
    - fedora-all-aarch64
    - fedora-all-x86_64
```

Reply:
Not sure if this is wanted. @pirat89 Do you think it's a good idea to have the copr targets moved automatically or do you want to retain manual control of them?

Also where can I find the aliases and their mapping? From packit [docs](https://packit.dev/docs/configuration/#available-copr-build-targets) `fedora-all` maps to `fedora-stable` + `fedora-development`. IIUC Fedora does not know anything like stable, only [supported releases (f33 + f34)](https://fedoraproject.org/wiki/Releases#Current_Supported_Releases) and [developmnet releases (f35 + rawhide)](https://fedoraproject.org/wiki/Releases#Development). If `fedora-stable` = supported releases and `fedora-development` = development releases then it does not correspond to [packit config](https://github.com/packit/packit/blob/main/packit/config/aliases.py#L34)...

Reply:
> IIUC Fedora does not know anything like stable, only [supported releases (f33 + f34)](https://fedoraproject.org/wiki/Releases#Current_Supported_Releases) and [developmnet releases (f35 + rawhide)](https://fedoraproject.org/wiki/Releases#Development). If `fedora-stable` = supported releases and `fedora-development` = development releases then it does not correspond to [packit config](https://github.com/packit/packit/blob/main/packit/config/aliases.py#L34)...

Your expectations are correct: `stable=supported` and `development=development`. The wording might not be accurate (will improve that), but I hope it can be understood. There are two problems with the `f35`:
* The mapping you've linked is a backup, we try to get this info from bodhi. The hardcoded one is used only when the API is not available. (Thanks for the reminder, we need to update that.)
* We also check, if the chroots/targets are available on `Copr` so it does not need to be available even when it's given to us by Bodhi.

Reply:
@vojtechsokol I think we could give it a try. In case we find a not-simple python issue with a newest fedora, we will just change it. It's better to know when leapp becomes broken for the newest python/system.

Reply:
@vojtechsokol

Here is an update of the documentation if you want to take a look: https://github.com/packit/packit.dev/pull/314
and here is the update of the backup values: https://github.com/packit/packit/pull/1315